bcda/cli.go

```diff
@@ -10,8 +10,33 @@ import (
 	"regexp"
 	"strconv"
 	"time"
+
+	"github.com/CMSgov/bcda-app/bcda/models"
 )
 
+func createACO(name, cmsID string) (string, error) {
+	if name == "" {
+		return "", errors.New("ACO name (--name) must be provided")
+	}
+
+	var cmsIDPt *string
+	// ACO ID is not required, but must match `AXXXX` if provided
```

Comment:
i think this comment is explained in the clear, readable code below and can be removed.  you know i have strong opinions about inline comments and find them distracting so if you want to keep it, that's cool too!

Reply:
👍 Hooray! I didn't know if it would be helpful to explain the business rule or not. Happy to remove it.